client/apps/shipping-label/view.js

```diff
@@ -239,8 +242,9 @@ class ShippingLabelRootView extends Component {
 	}
 
 	renderLabels() {
-		if ( ! _.every( this.props.shippingLabel.labels, 'statusUpdated' ) ) {
-			return <LoadingSpinner />;
+		if ( this.needToFetchLabelsStatus ) {
```

Comment:
The change in `master` was removing the `<Spinner>`. The correct action here is removing this conditional, since fetching the labels status is handled in `componentWillMount`. Fixed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/8877cf3ceb332e56afcbdd6258d7b8f7a3b61d4f